source/feature/openvswitch/native-openvswitch.html.md

```diff
@@ -46,6 +46,20 @@ In the following table, networking entities are listed with their implementation
 ### OVS networking model 
 ![OVS networking model](/feature/openvswitch/ovs_networking_model.svg)
 
+## Limitations
+
+Currently there are some limitations compared to Linux bridge [RFE OvS support](https://bugzilla.redhat.com/show_bug.cgi?id=1195208):
+
+- [Multiple gateways](https://bugzilla.redhat.com/show_bug.cgi?id=1383035) - Source routing is not implemented for the OVS switch type. This means that only management network can have gateway.
+- DHCPv6 - Currently not supported
+- [Port mirroring](https://bugzilla.redhat.com/show_bug.cgi?id=1362492)
+- [Host QoS](https://bugzilla.redhat.com/show_bug.cgi?id=1380271)
+- [Setting bridge options](https://bugzilla.redhat.com/show_bug.cgi?id=1380273)
+- [iSCSI Bond](https://bugzilla.redhat.com/show_bug.cgi?id=1441245)
+- LLDP support
+- Setting DNS
+- Upgrade from Linux bridge to OvS and vice versa
+- Direct connection of VM to the network - Only supported with OVN
```

Comment:
this is not a short-term limitation, it is an issue that would not change and deserves being mentioned in the Overview.

Reply:
Done